docs/components/KRate.md

```diff
@@ -0,0 +1,81 @@
+---
+title: KRate
+lang: en-US
+---
+
+# KRate
+
+Used for rating.
+
+## Install
+
+::: code-group
+
+```bash [pnpm]
+pnpm add @ikun-ui/rate
+```
+
+```bash [yarn]
+yarn add @ikun-ui/rate
+```
+
+```bash [npm]
+npm install @ikun-ui/rate
+```
+
+:::
+
+## Basic usage
+
+Use `colors` prop to set color for different level.
+
+<demo src="../../../../example/rate/basic.svelte"  github='Rate'></demo>
+
+## With text
+
+Using text to indicate rating score.
+
+Add attribute `showText` to display text at the right of Rate. You can assign `texts` for different scores using texts.
+
+<demo src="../../../../example/rate/with-text.svelte" github='Rate'></demo>
+
+## More icons
+
+You can use different icons to distinguish different rate components.
+
+You can customize `icons` by passing icons.
+
+<demo src="../../../../example/rate/more-icon.svelte" github='Rate'></demo>
+
+## Readonly
+
+Readonly Rate is for displaying rating score.
+
+<demo src="../../../../example/rate/readonly.svelte" github='Rate'></demo>
+
+## Rate Props
+
+| Name             | Type                                | Default                  | Description                                                                                     |
+| ---------------- | ----------------------------------- | ------------------------ | ----------------------------------------------------------------------------------------------- |
+| max              | `number`                            | `5`                      | max rating score.                                                                               |
+| value            | `number`                            | `0`                      | rate value.                                                                                     |
+| showScore        | `boolean`                           | `false`                  | whether to display current score. `showScore` and `showText` cannot be `true` at the same time. |
+| scoreTemplate    | `string`                            | `'{value}'`              | score template.                                                                                 |
```

Comment:
I thought it would be possible to add a doc example for scoreTemplate and showScore